cmake_tests/mock/MOCK_VARIABLES.md

```
@@ -0,0 +1,145 @@
+
+### Constants (CMDEF_VARS.cmake)
+
+| Variable | Value |
+|---|---|
+| `CMDEF_ARCHITECTURE` | `mock_arch` |
+| `CMDEF_DISTRO_ID` | `mock_distro` |
+| `CMDEF_DISTRO_VERSION_ID` | `0.0.0` |
+
+## Key Mock-to-Mock Dependency
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)